src/elements/LicenseIconography/LicenseIconography.vue

```diff
@@ -52,9 +61,17 @@
   export default {
     name: 'LicenseIconography',
```

Comment:
It's missing the `inheritAttrs` option set to `false` as described in the [docs](https://vuejs.org/v2/guide/components-props.html#Disabling-Attribute-Inheritance).